RDlargeroundcut_dark

Dapple27 opened the pull request:
:cl:
add: Bata larga de RD , morada y oscura añadida ( se puede encontrar en el loadout)
imageadd: Sprite para Bata larga de RD , morada y oscura
/:cl: 

Una bata común y corriente como las demas , solo que es para RD's y es mas oscura que la original
![image](https://user-images.githubusercontent.com/43283605/54730259-64b36c00-4b5e-11e9-91a6-f66e0b6c90c2.png)

Commit:
RDlargeroundcut_dark

Una bata común y corriente como las demas , solo que es para RD's y es mas oscura que la original

iRyzor commented:
Todo bien menos con la idea de meterlo dentro del locker del RD.

Dapple27 commented:
pero no se donde ponerlos

Tilk1 commented:
Los civiles tambien queremos ropita cool

Kotiro22 commented:
Puedes añadirlos al loadout como las otras, solo seria copiar y pegar el code de las mias y cambiar de nombre las cosas a las que usases en esa, es bastante simple, tampoco estoy de acuerdo en verla en el taquillero de RD.

Commit:
Fix

Se arregla error en batas y ahora se pueden encontrar solo en el loadout , y se añade una nota en el code del labcoat

Dapple27 commented:
Ya he arreglado el problema que tenia de suit y ahora se puede encontrar en el loadout

iRyzor commented:
Qué necesidad había de crear ese dm? ¿tiene algo distinto de lo que ya trae la original?
code/HISPANIA/game/objects/structures/crates_lockers/closets/secure/scientist.dm

Dapple27 commented:
no me habia fijado que  seguia ahi , se suponia que era donde iba la bata , antes de meterla en el loadout

Dapple27 committed:
Delete scientist.dm

Dapple27 commented:
arreglao
